wandb/apis/public.py

```diff
@@ -4852,14 +4890,14 @@ class ArtifactVersions(Paginator):
 
     def __init__(
         self,
-        client,
-        entity,
-        project,
-        collection_name,
-        type,
-        filters=None,
-        order=None,
-        per_page=50,
+        client: Client,
+        entity: str,
+        project: str,
+        collection_name: str,
+        type: str,
+        filters: Optional[Mapping[str, Any]] = None,
+        order: Optional[str] = None,
+        per_page: Optional[int] = 50,
```

Comment:
Is per_page Optional?

Reply:
Ehhh, technically yes in the GraphQL schema, but in practice I think I think it shouldn't be. Changed to `per_page: int`, thanks for calling this out!